Builds/Test.py

```diff
@@ -196,22 +196,21 @@ def run_tests(args):
             resultcode, lines = shell(executable, (testflag,))
 
             if resultcode:
-                print('ERROR:', *lines, sep='')
+                if not ARGS.verbose:
+                    print('ERROR:', *lines, sep='')
                 failed.append([target, 'unittest'])
                 if not ARGS.keep_going:
                     break
-            ARGS.verbose and print(*lines, sep='')
 
             if not ARGS.nonpm:
                 print('npm tests for', target)
                 resultcode, lines = shell('npm', ('test', '--rippled=' + executable,))
                 if resultcode:
-                    print('ERROR:\n', *lines, sep='')
+                    if not ARGS.verbose:
+                        print('ERROR:\n', *lines, sep='')
```

Comment:
If we are verbose, we've already printed it from within `shell`, so there's no point in repeating the entire output.